utils/src/main/kotlin/Utils.kt

```diff
@@ -391,6 +392,12 @@ fun JsonNode?.textValueOrEmpty(): String = this?.textValue()?.let { it } ?: ""
  */
 fun String.encodeOrUnknown() = fileSystemEncode().takeUnless { it.isBlank() } ?: "unknown"
 
+/*
+ * Convenience function for [JsonNode] that returns the an empty iterator
+ * if [JsonNode.fieldNames] is called on a null object or the field names otherwise.
```

Comment:
Add a comma before "or" for better readability.

Reply:
Done.